packages/ui/test/membership/modals/UpdateMembershipModal.test.tsx

```diff
@@ -58,6 +60,17 @@ describe('UI: UpdatedMembershipModal', () => {
   let batchTx: any
   let profileTxMock: jest.Mock
 
+  const useMyMemberships: MyMemberships = {
+    active: undefined,
+    members: [],
+    setActive: (member) => (useMyMemberships.active = member),
```

Comment:
This assignment to a `const` looks risky but seems to work somehow?

Reply:
On Javascript object `const` just means that the object can't be reassigned but (counterintuitively) the object properties can still be freely mutated (this is why lib like [immutable-js](https://github.com/immutable-js/immutable-js/) exist.

But if the object is actually not reassigned IMO `const` should be used because `let` would imply that somewhere `useMyMemberships` might be assigned a different object reference.